openwisp_monitoring/monitoring/base/models.py

```diff
@@ -703,15 +703,6 @@ class Meta:
             ('view_alertsettings_inline', 'Can view Alert settings inline'),
         )
 
-    def full_clean(self, *args, **kwargs):
-        if self.custom_threshold == self.config_dict['threshold']:
-            self.custom_threshold = None
-        if self.custom_tolerance == self.config_dict['tolerance']:
-            self.custom_tolerance = None
-        if self.custom_operator == self.config_dict['operator']:
-            self.custom_operator = None
-        return super().full_clean(*args, **kwargs)
-
```

Comment:
Why do we need to remove this?

Reply:
If we try to add a new `AlertSettings` with `operator (less than)` and let other fields remain empty, then we get an empty `AlertSettings form (see below)` without having default values. This is due to the fact that the **full_clean()** method sets fields to **None**. Therefore, I removed that method to prevent this behaviour.

**Note**: This behaviour is only seen when we choose the operator `< (less than)`.

https://user-images.githubusercontent.com/56113566/191241144-9e34390d-7449-4a0e-ad3c-2cb6b6dcc234.mp4